### PR TITLE
Localized error messages for jQuery Validation

### PR DESCRIPTION
**Pull request by ictbeheer**

### CODE REVIEW

**Comment by rilwis** [comment]
Thanks for your contribution.

I notice the code is different from jQuery Validation's Github repo: https://github.com/jquery-validation/jquery-validation/blob/master/src/localization/messages_ar.js

Would you mind sharing the source that you get the files from? Or can you please update to match the repo?

Thanks

File: inc/validation.php [hunk 1]
```diff
@@ -21,6 +21,14 @@ public function rules( RW_Meta_Box $object ) {
 	public function enqueue() {
 		wp_enqueue_script( 'rwmb-validation', RWMB_JS_URL . 'validation.min.js', [ 'jquery', 'rwmb' ], RWMB_VER, true );
 
+		$locale       = determine_locale();
+		$locale_short = substr( $locale, 0, 2 );
+		$locale       = file_exists( RWMB_DIR . "js/validation/i18n/messages_$locale.js" ) ? $locale : $locale_short;
+
+		if ( file_exists( RWMB_DIR . "js/validation/i18n/messages_$locale.js" ) ) {
+			wp_register_script( 'rwmb-validation-i18n', RWMB_JS_URL . "js/validation/i18n/messages_$locale.js", ['rwmb-validation'], RWMB_VER, true );
```
Comment (rilwis) [inline inc/validation.php:29]: Should we enqueue this script instead of just registering it?

**Review by rilwis — CHANGES_REQUESTED** [review]

File: inc/validation.php (hunk 1 above)
Comment (ictbeheer) [reply inc/validation.php:29]: Yes you are right. I see that this PR has already been merged. Should I create a new one for this fix?

**Comment by rilwis** [comment]
@ictbeheer Can you please create a PR again for this. @tuankiet2605 merged it wrongly into an incorrect branch. We need to merge it into `master`.

**Comment by ictbeheer** [comment]
> @ictbeheer Can you please create a PR again for this. @tuankiet2605 merged it wrongly into an incorrect branch. We need to merge it into `master`.

I've create a new PR:  #1525

**Comment by rilwis** [comment]
Thank you very much!

**Pull request merged**